htmlproofer/plugin.py

```diff
@@ -52,6 +52,8 @@ def bad_url(self, url_status):
             return False
         elif url_status == 503:
             return False
+        elif url_status == 999:
```

Comment:
@koenvervloesem could you please add some comments here to explain it ?

Reply:
Sure, done!

Reply:
Thanks, merging